lib/features/home/widgets/feed.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:fusecash/generated/i18n.dart';
+import 'package:fusecash/models/transactions/transaction.dart';
+import 'package:fusecash/models/transactions/transfer.dart';
+import 'package:fusecash/models/app_state.dart';
+import 'package:flutter/material.dart';
+import 'package:fusecash/features/home/widgets/transfer_tile.dart';
+import 'package:fusecash/redux/viewsmodels/feed.dart';
+
+class Feed extends StatefulWidget {
```

Comment:
Do we want to make this Stateless eventually?

Reply:
Yes